lib/sedna/algorithms/multi_task_learning/__init__.py

```diff
@@ -12,5 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .task_jobs import *
+import task_jobs
```

Comment:
For consistency, why we are not using from . import task_jobs like the other files did

Reply:
fix